index.html

```diff
@@ -5,9 +5,96 @@
 
     <link href="styles.css" rel="stylesheet" />
 
-    <title>Rocket Academy</title>
+    <title>Qian's Portfolio</title>
   </head>
   <body>
-    <h1>Hello, world!</h1>
+    <section class="header-section">
+      <div class="header">
+        <p id="author-name">Yap Qianling</p>
+        <div class="header-navigation">
+          <a href="#home">Home</a>
+          <a href="#projects">Projects</a>
+          <a href="#skills">Skills</a>
+          <a href="#contact">Contact</a>
+        </div>
+        <div>
+          <button class="hire-button">Contact Me ></button>
+        </div>
+      </div>
+
+    </section>
+
+    <section id="home" class="home-section">
+        
+      <div class="about">
+        <p class="title">
+          Hey there, I'm Qianling! <br />An aspiring Software Engineer.
+        </p>
+        <p>
+          Based in Singapore, I am passionate about creating a product from
+          scratch, and bringing it to life.
+  
+         
+        </p>
+        <div>
+          <button class="say-hello-button">Let's Start ></button>
```

Comment:
You could try seeing if we need the `<div>` without much of a use. Can we make in-line elements to block elements? :)